.github/ISSUE_TEMPLATE.md

```diff
@@ -1,4 +1,22 @@
-Which operating system does the issue occur on?
+For bug reports, the following information can optionally help speed up the process:
 
```

Comment:
I liked the guidance in the old template for specifying the difference between the expected behavior, since people (myself included) May easily assume the expected behavior is different from each other. This is especially true with complex interactions of shells and other applications.

Reply:
You mean what I removed from the CHANGELOG?

Most of the stuff I deleted because I thought it would be obvious, but asking what the expected behavior is is a really good idea, thanks.

Reply:
I was refering to these sections from the `CONTRIBUTING.md`:

```
# Behavior
[Description of Alacritty's current behavior]

# Expectation
[Description of expected behavior]
```

However, I'd turn these into a leading sentence somehow, not explicit sections.

Reply:
`What have you observed happening? What do you expect to happen?`